static/css/base.css

```
@@ -1,14 +1,15 @@
-/* # Copyright (c) Liam Suorsa and Mika Suorsa */
+/* # Copyright (c) Liam Suorsa and Mika Suorsa */
 :root {
     --font-base: 'Inter', 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif;
```

Stylelint (17.11.1)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)